dev/src/generate/binding_python.rs

```diff
@@ -17,9 +17,87 @@
 
 use crate::generate::parser::Services;
 use anyhow::Result;
+use itertools::Itertools;
+use std::fs;
+use std::path::PathBuf;
 
-pub fn generate(services: &Services) -> Result<()> {
-    println!("{:?}", services);
+use super::parser::ConfigType;
+
+/// TODO: add a common utils to parse enabled features from cargo.toml
+fn enabled_service(srv: &str) -> bool {
+    match srv {
+        // not enabled in bindings/python/Cargo.toml
+        "etcd" | "foundationdb" | "ftp" | "hdfs" | "rocksdb" | "tikv" => false,
+        _ => true,
+    }
+}
+
+pub fn generate(project_root: PathBuf, services: &Services) -> Result<()> {
+    let mut s = fs::read_to_string(project_root.join("dev/templates/python"))
+        .expect("failed to read python template file");
+
+    for (srv, config) in services.clone().into_iter() {
+        if !enabled_service(srv.as_str()) {
+            continue;
+        }
+
+        s.push_str("\n    @overload\n");
```

Comment:
I'm thinking of using https://crates.io/crates/handlebars instead of building string directly.

Reply:
askama seems cool but it's latest release is one year ago, I have a bit concern about this.

Reply:
Also https://github.com/rinja-rs/askama/issues/1035

Reply:
I'd use https://github.com/mitsuhiko/minijinja (but IMO any one of them is fine choice as long as the templating crate works and is actively maintained).

Reply:
Cool, let's do this.